lib/main_publiccloud.pm

```diff
@@ -233,6 +239,17 @@ sub load_publiccloud_tests {
             load_maintenance_publiccloud_tests();
         } elsif (get_var('PUBLIC_CLOUD_HIMMELBLAU')) {
             loadtest('publiccloud/himmelblau');
+        } elsif (get_var('PUBLIC_CLOUD_APP_IMG')) {
+            my $publiccloud_app_img = get_required_var('PUBLIC_CLOUD_APP_IMG');
+            if (exists($publiccloud_app_img_tests{$publiccloud_app_img})) {
```

Comment:
if you use here `get_required_var()`, no need to the if-then and the die anymore :)